### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by JesseCrocker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
 # StaticMaps
-Static Map renderer using mapnik
+Static Map rendering service using mapnik. Post GeoJSON, get back a map rendered on a background.
+Written in python, using django.
+Designed to be used as a micro service and deployed as a docker container.
+
+Prebuilt docker images available as gaiagps/staticmapservice, If you want to add your own background, extend that image. 
+
+For an example service file for deploying with fleet see staticmaps@.service
```
Comment (andrewljohnson) [inline README.md]: This ends up showing weird on github.com: staticmaps@.service

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
 # StaticMaps
-Static Map renderer using mapnik
+Static Map rendering service using mapnik. Post GeoJSON, get back a map rendered on a background.
```
Comment (andrewljohnson) [inline README.md:2]: This is fine, but when you check it in I'll propose a new PR.

I would organize into headings and make the first sentence complete and forceful

File: README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+Written in python, using django.
+Designed to be used as a micro service and deployed as a docker container.
+
+Prebuilt docker images available as gaiagps/staticmapservice, If you want to add your own background, extend that image. 
+
+For an example service file for deploying with fleet see staticmaps@.service
+
+## API
```
Comment (andrewljohnson) [inline README.md:10]: I've always wondered if there is a standard for documenting APIs, and is this it?

Comment (JesseCrocker) [inline README.md:10]: There's a couple standards, and no one outside of giant companies uses them

File: README.md [hunk 4]
```diff
@@ -1,2 +1,74 @@
...
+    
+    *  Polygons are stroked using the same styling options as LineString features
+    * `fill` fill color, default is `red`
+    * `opacity` fill opacity, default is 0.6
+
+See [Mapnik LineSymbolizer reference](https://github.com/mapnik/mapnik/wiki/LineSymbolizer) and [Mapnik PolygonSymbolizer reference](https://github.com/mapnik/mapnik/wiki/PolygonSymbolizer) for more info.
+
+### GET or POST /{bounds}/{width}x{height}/{background}.{format}
+#### Parameters
+* bounds: A bounding box in e,s,w,n format. Example `-120,38,-119,39`. Bounding boxes that cross the anti-meridian are not currently supported
```
Comment (andrewljohnson) [inline README.md]: This doesn't look like e,s,w,n to me

**Comment by andrewljohnson** [comment]
lgtm, small writeup on how we use this?

**Pull request merged**